### ISSUE TITLE
Porntrex error

### ISSUE THREAD

**Issue by tetzy** [dobbelina/repository.dobbelina#536]
As of last night, porntrex is down and reports the following error:

![z](https://user-images.githubusercontent.com/59111079/120934822-21049880-c6bd-11eb-801d-6346746163d2.png)

![zz](https://user-images.githubusercontent.com/59111079/120934832-2bbf2d80-c6bd-11eb-9367-f571b3c05e6c.png)

If you need a more detailed log, I'll happily provide. 

thanks.

**Comment by ErosVece** [comment]
What version of Cumination or you using?

**Comment by imurreflection** [comment]
No issue. The site is working fine. Please use VPN

**Comment by tetzy** [comment]
Jesus - I'm banging my head against a wall here.

I am really struggling with the change made to the favorites menu in cumination-1.1.14 (moving "Remove from favorites" from the top of the menu to the middle of the menu) so I downgraded from cumination-1.1.16 back to cumination-1.1.13 (the last most recent version with the old menu) and it's the version reporting the error. 

I know it's a petty thing, but I use "remove from favorites" all the time (I delete titles as soon as I've watched them) and I just cant get used to that change. My kingdom for the ability to sort that menu manually.
